src/lib/y2security/security_policies/bootloader_password_rule.rb

```diff
@@ -29,15 +29,20 @@ def initialize
 
         # DISA STIG defines a rule for UEFI (SLES-15-010200) and another rule for non-UEFI
         # (SLES-15-010190). The condition to check both rules in YaST is exactly the same, so let's
-        # simply adapt the rule ID according to the system type.
-        id = Y2Storage::Arch.new.efiboot? ? "SLES-15-010200" : "SLES-15-010190"
-
-        super(
-          id,
-          # TRANSLATORS: security policy rule
-          _("Bootloader must be protected by password and menu editing must be restricted"),
-          :bootloader
-        )
+        # simply adapt the rule name and ID according to the system type.
+        if Y2Storage::Arch.new.efiboot?
+          name = "grub2_uefi_password"
+          id = "SLES-15-010200"
+        else
+          name = "grub2_password"
+          id = "SLES-15-010190"
+        end
+
+        # TRANSLATORS: security policy rule
+        description = _("Bootloader must be protected by password and menu editing must be " \
```

Comment:
np: I would use "boot loader" (I have checked that we use that form rather often in our UI).